web-client/src/main.ts

```diff
@@ -18,6 +18,22 @@ tf.ready()
 
 // create vue app
 const app = createApp(App)
+
+// Global error handler
+app.config.errorHandler = (err, instance, info) => {
+  if (err instanceof TypeError) {
+    // Implementation bug
+    const toaster = createToaster({ duration: 5000 })
+    toaster.error('Sorry, something went wrong on our side. Please reach out on slack.')
+    console.error(err, instance, info)
+  } else {
+    // Unknown error
+    const toaster = createToaster({ duration: 5000 })
+    toaster.error('Something went wrong. Please try again later or reach out on slack.')
+    console.error(err, instance, info)
+  }
```

Comment:
hoo, that's a good idea, silent failures are the worst!

you can also commonize some code, but that's nitpicking
```suggestion
  const toaster = createToaster({ duration: 5000 })
  if (err instanceof TypeError) {
    // Implementation bug
    toaster.error('Sorry, something went wrong on our side. Please reach out on slack.')
  } else {
    // Unknown error
    toaster.error('Something went wrong. Please try again later or reach out on slack.')
  }
  console.error(err, instance, info)
```